backend/Recodo.BLL/Services/MailService.cs

```diff
@@ -0,0 +1,28 @@
+using MimeKit;
+using MimeKit.Text;
+
+namespace Recodo.BLL.Services
+{
+    public sealed class MailService
+    {
+        public void SendEmail(string body, string email, string name = "")
+        {
+            MimeMessage message = new MimeMessage();
+            message.From.Add(new MailboxAddress("Recodo", "recodo.app.22@gmail.com"));
```

Comment:
If you've registered this account, add it to appsettings to be reused in other places and extract via IConfiguration in this class